plugins/node/opentelemetry-instrumentation-pino/README.md

```diff
@@ -30,7 +30,14 @@ registerInstrumentations({
     new PinoInstrumentation({
       // Optional hook to insert additional context to log object.
       logHook: (span, record, level) => {
-        record['resource.service.name'] = provider.resource.attributes['service.name'];
+        record['resource.service.name'] =
+          provider.resource.attributes['service.name'];
+      },
+      // Log span context under custom keys
```

Comment:
Please make it more clear that this is optional.
Besides that the text below which describes the added fields should be improved to mention that below values are the defaults.

Reply:
Updated, let me know if you think that's clear or needs to be updated still